wpilibj/src/athena/java/edu/wpi/first/wpilibj/IterativeRobot.java

```diff
@@ -74,10 +66,10 @@ public void startCompetition() {
     // Tell the DS that the robot is ready to be enabled
     FRCNetworkCommunicationsLibrary.FRCNetworkCommunicationObserveUserProgramStarting();
 
-    // loop forever, calling the appropriate mode-dependent function
+    // loop forever, calling the appropriate mode-dependent m_function
     LiveWindow.setEnabled(false);
     while (true) {
-      // Call the appropriate function depending upon the current robot mode
+      // Call the appropriate m_function depending upon the current robot mode
```

Comment:
Here and line 69, erroneous replacement of "function" with "m_function" in comment.